content/docs/start/experiments.md

```diff
@@ -21,39 +21,34 @@ the [`example-dvc-experiments`][ede] project.
 
 <details>
 
-### ⚙️ Installing the example project
+### ⚙️ Initializing a project with DVC experiments
 
-These commands are run in the [`example-dvc-experiments`][ede] project. You can
-run the commands in this document after cloning the repository, installing the
-requirements, and pulling the data.
+If you already have a DVC project, that's great. You can start to use `dvc exp`
+commands right away to run experiments in your project. (See the [user's guide]
+for detailed information.) Here, we briefly discuss how to structure an ML
+project with DVC experiments using `dvc exp init`.
 
-#### Clone the project and create virtual environment
+[user's guide]: /doc/user-guide/experiment-management/
 
-Please clone the project and create a virtual environment.
-
-> We strongly recommend to create a virtual environment to keep the libraries we
-> use isolated from the rest of your system. This prevents version conflicts.
+A typical machine learning project has data, a set of scripts that train a
+model, a bunch of hyperparameters that tune training and models, and outputs
+metrics and plots to evaluate the models. DVC makes certain assumptions about
```

Comment:
one more thing reading this. "DVC makes assumptions" is too strong (like DVC is an opinionated framework while it's not). It's only `dvc exp init` has defaults, right?

Reply:
Revised that phrase. Thank you. @shcheklein PTAL.